tests/integration/common/checks-after-conversion/main.fmf

```diff
@@ -34,6 +34,10 @@ order: 52
         test: pytest -m test_correct_distro
 
     /grub_default:
+        adjust+:
+          enabled: false
+          when: distro == stream-9-latest
+          because: The versions of latest RHEL kernel < latest Stream kernel, which is not handled very well in the code.
```

Comment:
Do we have some jira ticket to link here?

Reply:
It's this one https://issues.redhat.com/browse/RHELC-1717
I'll attach it in the comment.

Reply:
Yes please attach that there. Maybe even use the keyword `link: https://issues.redhat.com/browse/RHELC-1717`